autonomy/replay/agent.py

```diff
@@ -99,9 +101,18 @@ def stop(
         self,
     ) -> None:
         """Stop the process."""
-        self.agent_dir.cleanup()
+
         os.chdir(str(self.cwd))
+        with contextlib.suppress(OSError, PermissionError, FileNotFoundError):
+            shutil.rmtree(str(self.agent_dir))
```

Comment:
This was causing some failures on windows

Reply:
yes because of permissions. We can fix this later, maybe worth a ticket

Reply:
@angrybayblade ticket please